images/bind9/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster-slim
```

Comment:
mmm, this would come from docker.io, which will fail in CI

Reply:
are we able using an image from quay.io, debian-based to avoid changing the other layers below, or ubi9/cs9-based if you have the patience to make it.

Reply:
yes, I will move it to centos 9